src/site/content/en/fast/optimize-css-background-images-with-media-queries/index.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: post
+title: Optimize CSS background images with media queries
+authors:
+  - demianrenzulli
+description: |
+  Use media queries to send images that are only as large as they need to be, 
+  a technique commonly known as responsive images.
+date: 2020-03-05
+updated: 2020-03-05
+---
+
+Many sites request heavy resources, like images, that are not optimized for certain screens, and send large CSS files containing styles that some devices will never use. Media queries are a popular technique for delivering tailored stylesheets and assets to different screens to reduce the amount of data transferred to users and improve page load performance. This guide shows you how to use media queries to send images that are only as large as they need to be, a technique commonly known as **responsive images**.
+
+## Prerequisites
+
+This guide assumes that you're familiar with [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools). You can use another browser's DevTools instead if you prefer. You'll just need to map the Chrome DevTools screenshots in this guide back to the relevant features in your browser of choice.
+
+## Understand responsive background images
+
+First, analyze the network traffic of the unoptimized demo:
+
+1. Open the [unoptimized demo](https://use-media-queries-unoptimized.glitch.me/) in a new Chrome tab.
+{% Instruction 'devtools-network', 'ol' %}
+{% Instruction 'reload-page', 'ol' %}
+
+{% Aside %}
+Check out [Inspect Network Activity With Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/network/) if you need more help with DevTools.
+{% endAside %}
+
+You'll see that the only image that's being requested is `background-desktop.jpg`, which has a size of **1006KB**:
+
+<figure>
+  <img src="./cdt-background-unoptimized.png"
+       alt="DevTools network trace for the unoptimized background image." class="w-screenshot">
+</figure>
+
+Resize the browser window and notice that the Network Log isn't showing any new requests being made by the page. This means that the same image background is being used for all screen sizes.
+
+You can see the styles that control the background image in [style.css](https://use-media-queries-unoptimized.glitch.me/style.css):
+
+```css
+body { 
+  background-position: center center; 
+  background-attachment: fixed; 
+  background-repeat: no-repeat; background-size: cover; 
+  background-image: url(images/background-desktop.jpg); 
+}
+```
+
+Here's the meaning of each of the properties used:
+
+- `background-position: center center:` Center the image vertically and horizontally.
```

Comment:
```suggestion
- `background-position: center center`: Center the image vertically and horizontally.
```
Need to move the last colon outside of the inline code because inline code formatting implies verbatim code that you can copy-paste, and in this case it will create invalid syntax.